src/Module.Server/Modes/Conquest/CrpgConquestServer.cs

```diff
@@ -417,10 +434,25 @@ private void RewardUsers()
     {
         if (_rewardTickTimer!.Check(reset: true))
         {
+            int defenderMultiplierGain;
+            int attackerMultiplierGain;
+            if (_isOddRewardTick)
+            {
+                defenderMultiplierGain = 0;
+                attackerMultiplierGain = 0;
+                _isOddRewardTick = false;
+            }
+            else
+            {
+                defenderMultiplierGain = 1;
+                attackerMultiplierGain = -1;
+                _isOddRewardTick = true;
+            }

```

Comment:
It sounds like you could just double the tick interval right?

Reply:
Fips and i felt that it would feel better for a casual mode to get rewards more often. Of course it would be the same to get the tick each minute, but this felt better to me.

Reply:
Hmm, not a big fan of having inconsistent rewards between ticks. I think that will just confuse people.

Reply:
You think it's confusing? I think it's the same as finding out how the multi system even works on conquest. IMO it's not much more complex than it already is. I really think that more ticks are more fun than big ticks per minute.
But if you think that is too much i can remove it again.

Reply:
Ah nvm I misunderstood the code, I thought the even tick could be less than the odd tick for defenders.